### DISCUSSION TITLE
“Local” authentication rate limiting

### DISCUSSION THREAD

**Discussion by aidenmitchell** [requarks/wiki#4187] [Help / Questions] [answered]
Hi there,

Is there a rate limit on the "Local" authentication strategy? What prevents someone from brute-forcing it?

**Comment by NormandoHall** [comment]
Seams no yet

**Comment by Phil-hostwiki** [comment, marked as answer]
After 5 failed attempts, you get rate-limited with the message: "GraphQL error: Too many requests, please try again in 59 seconds..". 

![wikijs_rate_limit](https://user-images.githubusercontent.com/82508256/138612199-2a594c08-305e-4da5-90f3-4dfddacf3c18.jpg)

**Comment by ieaster1** [comment]
The above is correct.  Rate limits are not user configurable at the moment but they do exist.  You could change them within your environment if needed.

See:
https://github.com/Requarks/wiki/blob/f55caab359ca7715012e3f9a3b9e31f16cccf399/server/graph/schemas/authentication.graphql#L45
